libcontainer/integration/exec_test.go

```diff
@@ -534,12 +534,12 @@ func testPids(t *testing.T, systemd bool) {
 	config.Cgroups.Resources.PidsLimit = -1
 
 	// Running multiple processes, expecting it to succeed with no pids limit.
-	_ = runContainerOk(t, config, "/bin/sh", "-c", "/bin/true | /bin/true | /bin/true | /bin/true")
+	runContainerOk(t, config, "/bin/sh", "-c", "/bin/true | /bin/true | /bin/true | /bin/true")
```

Comment:
Doesn't this fall under the same umbreall than: https://github.com/opencontainers/runc/pull/4686/files#r2008449376 ?

Reply:
Right, thanks!